delete.go

```diff
@@ -43,21 +61,17 @@ status of "ubuntu01" as "stopped" the following will delete resources held for
 		if err != nil {
 			return err
 		}
-		switch s {
-		case libcontainer.Stopped:
-			destroy(container)
-		case libcontainer.Created:
-			container.Signal(syscall.SIGKILL)
-			for i := 0; i < 100; i++ {
-				time.Sleep(100 * time.Millisecond)
-				if err := container.Signal(syscall.Signal(0)); err != nil {
-					destroy(container)
-					return nil
-				}
+		if context.Bool("force") {
+			killContainer(container)
```

Comment:
On Tue, Jun 28, 2016 at 07:51:47PM -0700, Qiang Huang wrote:
> > +		if context.Bool("force") {
> > +			killContainer(container)
>
> Looks weird to send signal to a stopped container, though it won't
> cause any errors.

Unless you hit #871 (which I think still needs to be fixed [1]).

[1]: https://github.com/opencontainers/runc/issues/871#issuecomment-225686558